src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+html body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  --text-active: hsl(193, 100%, 43%);
+}
```

Comment:
1. Not sure why you refer to body selector like this `html body`, it can be just body
2. CSS variables and global font styles are usually written inside `:root` or `html` selectors (this is just a common practice not an obligation).